### PR TITLE
Update User Guide for new features in FitnessMotivator

### PR DESCRIPTION
**Pull request by Daviancold**

### CODE REVIEW

File: src/main/java/fitness/goals/ExerciseGoalList.java [hunk 1]
```diff
@@ -123,14 +117,13 @@ public ExerciseGoal newExercise(String[] parameters) {
...
      * @param exercise An Exercise object to be converted and added into the list
      * */
-    @Override
-    public void add(Exercise exercise) {
+    public void add(Exercise exercise, boolean isDone) {
         goals.add(new ExerciseGoal(
                 exercise.getExerciseName(),
                 exercise.getType(),
                 exercise.getSets(),
                 exercise.getReps(),
```
Comment (genexus85) [inline src/main/java/fitness/goals/ExerciseGoalList.java:125]: I like how you made a separate add method to improve readability!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -866,6 +869,150 @@ Arms: Tricep Dips, 8 sets & 10 reps
...
+```
+fitness goal
+fitness goal new
+fitness goal [INDEX]
+```
+
+* The `fitness` and `goal` are case-sensitive.
+* Running the command with no parameters will generate different exercises goals with each repeated command.
+* Use the `fitness goal` command to find out the indexes of the respective exercises goals.
+* Only `new` and `[INDEX]` are accepted as parameters, where `[INDEX]` is an integer value. The parameters are optional as well.
```
Comment (genexus85) [inline docs/UserGuide.md:917]: I think it will be good to explain the purpose of using `fitness goal new` in comparison to just `fitness goal`

**Review by genexus85 — COMMENTED** [review]
Overall, looks good to me! just a minor suggestion to improve the DG.

File: docs/UserGuide.md (hunk 2 above)
Comment (Daviancold) [reply docs/UserGuide.md:917]: Thank you for your suggestion! I have made the changes.

**Pull request merged**